roles/shell/tasks/alacritty.yml

```diff
@@ -1,12 +1,14 @@
 ---
 - name: Get URL pointing to the latest alacritty .deb
-  # Ignore linter warning abot wget - it's cleaner than the alternative.
-  args:
-    warn: false
   shell: wget --quiet https://github.com/jwilm/alacritty/releases -O- | grep "_amd64.deb" | grep release | cut -d '"' -f2 | head -1 | sed "s/^/https:\/\/github.com/"
   register: alacritty_latest_deb_url
-  failed_when: alacritty_latest_deb_url.rc > 0
   changed_when: false
+  ignore_errors: yes  # Use ignore_errors instead of failed_when
```

Comment:
Niyusha jan, would you please what is the difference has been made by this change? This is not that obvious for me.

Reply:
If our task cannot retrieve the "alacritty_latest_deb_url" we can find it out with a more clear message than just failing a task named "**Get URL pointing to the latest alacritty .deb**".
By the way I think it is unnecessary to split this task into two.